openshift/dynamic/apply.py

```diff
@@ -183,6 +183,62 @@ def list_merge(last_applied, actual, desired, position):
         return desired
 
 
+def recursive_list_diff(list1, list2, position=None):
```

Comment:
Should we be calling these functions from the apply/merge functions?

Reply:
I wasn't hugely happy with the similarity of the list_merge/list_diff code, but I wasn't sure that I could eliminate any duplicated code through use of the diff functions. Can you see a mechanism to do so?